app/_src/production/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: About Kuma in Production 
+content_type: explanation
+subtitle: Deploying Kuma in a production environment. 
+---
+
+After you've completed your initial test and assessment of {{site.mesh_product_name}}, it's time to deploy {{site.mesh_product_name}} in your production environment. Before doing so, it's important to understand the overall steps in the deployment process as well as the different options you have when deploying {{site.mesh_product_name}}. This guide walks you through the big picture steps and options so you can make the right decisions when it comes to deployment time.
+
+## Overview of deployment steps
+
+Deploying {{site.mesh_product_name}} to a production environment involves the following steps:
+
+1. [Decide which deployment topology you plan to use](#deployment-topologies).
+1. [Install `kumactl`](#kumactl).
+1. [Deploy the control plane](#control-plane-and-data-plane-architecture).
+1. [Configure the data plane](#control-plane-and-data-plane-architecture).
+1. [Configure security features for {{site.mesh_product_name}}](#{{site.mesh_product_name}}-security).
+
+### Deployment topologies
+
+There are two deployment models that can be adopted with {{site.mesh_product_name}}: standalone and multi-zone. You can use these modes to address any service mesh use case, including:
+* A simple model with the service mesh running in one zone
+* A more complex model where multiple Kubernetes or VM zones are involved
+* A hybrid universal model where {{site.mesh_product_name}} runs simultaneously on Kubernetes and VMs
+
+The following table describes some common use cases and the deployment modes you can use for them:
+
+| Use case | Recommended deployment mode |
+| -------- | --------------------------- |
+| You want to migrate from on-premise or virtual machines to the cloud in a brownfield project. | Multi-zone |
+| You only intend to deploy {{site.mesh_product_name}} in one zone, like one Kubernetes cluster or Amazon VPC. | Standalone |
+| You want to run a mix of Kubernetes and Universal zones. | Multi-zone |
+| You want to run workloads in different regions, clouds, and/or datacenters. | Multi-zone |
+
+### kumactl
+
+The first step after you pick your deployment mode is to install `kumactl`. `kumactl` is a CLI tool that you can use to access the {{site.mesh_product_name}}. It can do the following:
```

Comment:
```suggestion
The first step after you pick your deployment mode is to install `kumactl`. `kumactl` is a CLI tool that you can use to access {{site.mesh_product_name}}. It can do the following:
```